src/helpers/types/state.ts

```diff
@@ -62,6 +63,7 @@ export interface IAppState {
 }
 
 export interface IRootState {
+    offline?: any;
```

Comment:
Is this really `any`?

Reply:
Unfortunately, yes. (https://github.com/kickstartcrew/redux-offline-queue/blob/main/typings.d.ts)

Reply:
I don't understand. Those "any" are because the user state or actions are in fact unknown. It seems to me that this has this reducer https://github.com/kickstartcrew/redux-offline-queue/blob/394307f2cfbf21ac22ab194c4b05f7ac8dd05045/src/reducer.js#L18 with this type https://github.com/kickstartcrew/redux-offline-queue/blob/394307f2cfbf21ac22ab194c4b05f7ac8dd05045/src/initialState.js#L7
Can we somehow create the type?

Reply:
Fixed.